### PR TITLE
Fixed for users from bitbucket

### PR DESCRIPTION
**Pull request by borek2**
BB users couldnt use the app, now they can. Userinfo problems and the situation that there is an end event without a start event

### CODE REVIEW

File: src/resolvers/userResolver.js [hunk 1]
```diff
@@ -15,6 +15,10 @@ function UserResolver(platform) {
                 });
             } else {
                 //Bitbucket requires authentication, nogo here
+                user.userInfo = {
+                    "picture": '../resources/anonymous.png',
```
Comment (DaanvanderValk) [inline src/resolvers/userResolver.js]: I don't see this file included in this branch, were is it located?

Comment (borek2) [inline src/resolvers/userResolver.js]: @agudek added this some time ago for users without a picture

Comment (DaanvanderValk) [inline src/resolvers/userResolver.js]: Nevermind

Comment (agudek) [inline src/resolvers/userResolver.js]: It can be found in TI2806/resources/

**Pull request merged**